src/main/java/book/store/model/Book.java

```diff
@@ -0,0 +1,32 @@
+package book.store.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "books")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @NotNull
+    private String title;
+    @NotNull
+    private String author;
+    @NotNull
+    @Column(unique = true)
+    private String isbn;
+    @NotNull
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
```

Comment:
better use @Column for **each** field. Specify the column name there. And use **unique** and **nullable** params where you need it.